mycroft/messagebus/client/ws.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, host=client_config.get("host"),
                  port=client_config.get("port"),
                  path=client_config.get("route"),
                  ssl=str2bool(client_config.get("ssl"))):
+        if not host:
+            # By default open localhost if host is empty, or None
+            host = "127.0.0.1"
```

Comment:
I didn't use `.get()` That will can return an empty string if that's the value, and `None` if it's not defined at all.  So intentionally I did a test for falseness.

Example:
```
>>> d = { "test":"" }
>>> d.get("test", "default")
''
```